demo/kserve/scripts/install/kserve-install.sh_old

```diff
@@ -0,0 +1,251 @@
+#!/bin/bash
```

Comment:
why keep it?
we can still refer it on previous branches.

Reply:
I accepted this. removed the file